control/control_performance_analysis/README.md

```diff
@@ -65,7 +65,7 @@ Error acceleration calculations are made based on the velocity calculations abov
 | `longitudinal_error_acceleration`          | float | [m / s^2]                                                                                                         |
 | `heading_error`                            | float | [rad]                                                                                                             |
 | `heading_error_velocity`                   | float | [rad / s]                                                                                                         |
-| `control_effort_energy`                    | float | [u * R * u^T]                                                                                                     |
+| `control_effort_energy`                    | float | [u *R* u^T]                                                                                                       |
```

Comment:
Will fix this part in here:
- https://github.com/autowarefoundation/autoware.universe/pull/9552